packages/core/src/customer/customer-request-options.ts

```diff
@@ -150,5 +150,10 @@ export interface BaseCustomerInitializeOptions extends CustomerRequestOptions {
  *
  */
 export interface ExecutePaymentMethodCheckoutOptions extends CustomerRequestOptions {
+    checkoutPaymentMethodExecuted?(data?: CheckoutPaymentMethodExecutedOptions): void;
```

Comment:
Is this a bolt specific propery?

Reply:
Currently it uses only for Bolt, because only Bolt has such functionality, and request such analytics event. 
If in feature some other payment method will need such callback it can be reused as well.